### ISSUE TITLE
Create a reset database command safeguarded by a password

### ISSUE THREAD

**Issue by ThomazFB** [the-orange-team/orange-bot#8]

**Comment by ThomazFB** [comment]
I'm tackling this right now using `crypto-js` and doing the following to match the password:

``` typescript
       const storedHash = process.env.RESET_HASH;
       const digestedArg = Base64.stringify(sha256(command.text));
       if (storedHash === digestedArg) {
            //proceed with database reset
       }
```

**Comment by ThomazFB** [comment]
Issue solved starting from: ed2722f107f7e239b6867ffe43b62c0cfca85db6
